src/proto/tagpost.proto

```diff
@@ -6,18 +6,71 @@ option java_multiple_files = true;
 option java_package = "com.google.tagpost";
 option java_outer_classname = "TagPostProto";
 
+import "google/protobuf/timestamp.proto";
+
 service TagpostService {
```

Comment:
Suggest to move the full RPC service, including itself and requests/responses to another file, and keep this file only contain Tag, Thread and Comment.